src/components/views/dialogs/DMInviteDialog.js

```diff
@@ -48,8 +74,46 @@ class DMRoomTile extends React.PureComponent {
         this.props.onToggle(this.props.member.userId);
     };
 
+    _highlightName(str: string) {
+        if (!this.props.highlightWord) return str;
+
+        // First encode the thing to avoid injection
+        str = htmlEntitiesEncode(str);
```

Comment:
It's important to defend against injection attacks, but React should already be encoding all strings as entities... Does the behaviour change with and without this present?

Reply:
I think this is mostly a sudden onset of paranoia based on zero technical footing, and therefore can be removed. Will test to make sure.

Reply:
Can you tell I tested this thoroughly? 

![image](https://user-images.githubusercontent.com/1190097/71842291-8a264d80-307e-11ea-8597-ff2605e03c5e.png)
![image](https://user-images.githubusercontent.com/1190097/71842296-8c88a780-307e-11ea-9fcc-d05cc5c2d9e4.png)

After removing the htmlentities bit:
![image](https://user-images.githubusercontent.com/1190097/71842310-93afb580-307e-11ea-9967-e60649121384.png)